### PR TITLE
WIP : remaing docs

### PR DESCRIPTION
**Pull request by tlyon3**
Adds the remaining docs needed for #2

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
👋 @tlyon3 thanks for all the hard work on these!

One key thing about these docs which you seem to be already doing but I want to highlight the importance of, is that **_every example should be independently runnable just by copy-pasting the one example_** without relying libraries or variables that were created higher up in the file.

This really helps users figure out what's what and helps us write compact, reproducible examples. As I said, I think you're already doing this but I wanted to strongly encourage you to keep doing it please :) and if there are any places where you know you've done things differently, we should try to go back and rejig things to meet this standard if possible.

Thanks again!

cc @sglyon

**Comment by sglyon** [comment]
Thanks @tlyon3 

The axes docs aren't building for me. Taking a quick look I noticed that there are some python code snippets in Julia blocks. Also, let's delete the bottom block that talks about `matches`

File: julia/axes.md [hunk 1]
```diff
@@ -0,0 +1,833 @@
...
+
+plot(trace, layout)
+```
+
+##### Set axis title text with Graph Objects
+
+Axis titles are set using the nested `title` property of the x or y axis. Here is an example of creating a new figure and using `Layout`, with magic underscore notation, to set the axis titles.
+
+```julia
+import plotly.express as px
```
Comment (sglyon) [inline julia/axes.md:138]: I'll fix it -- but this import statement is a python remnant

File: julia/axes.md [hunk 2]
```diff
@@ -0,0 +1,833 @@
...
+df = dataset(DataFrame, "iris")
+
+layout = Layout(
+    xaxis_ticks="inside",
+    yaxis_ticks="index",
+    yaxis_col=1
+)
+plot(df, x=:sepal_width, y=:sepal_length, facet_col=:species, kind="scatter", mode="markers",layout)
+
+fig.show()
```
Comment (sglyon) [inline julia/axes.md:218]: another pythonism

**Comment by sglyon** [comment]
Looks like there are more issues we need to fix. I did a quick search for "plotly express", "plotly.express", or "fig.show" and got many results:

```
plotlyjs.jl-docs on  issue-docs [$!?] via ஃ v1.6.1 via 🐍 v3.8.5 on ☁️  spencer.lyon@valorumdata.com 
❯ rg -S 'plotly express' ./julia
./julia/axes.md
177:Here is an example that configures the font family, size, and color for the axis titles in a figure created using Plotly Express.
204:Here is an example of turning on inside x-axis and y-axis ticks in a faceted figure created using Plotly Express. Note how the `yaxis_col` argument to `Layout` is used to only turn on the y-axis ticks for the left-most subplot.
225:Here is an example of updating the y-axes of a figure created using Plotly Express to display approximately 20 ticks.
253:Here is an example of updating the y axis of a figure created using Plotly Express to position the ticks at intervals of 0.5, starting at 0.25.
280:Here is an example of setting the exact location of ticks on the y axes of a figure created using Plotly Express.
307:Here is an example of enabling and styling the tick marks of a faceted figure created using Plotly Express. Note how the `col` argument to `yaxis attr` is used to only turn on and style the y-axis ticks for the left-most subplot.
332:Here is an example of disabling tick labels in all subplots for a faceted figure created using Plotly Express.
357:Here is an example of rotating the x-axis tick labels by 45 degrees, and customizing their font properties, in a faceted histogram figure created using Plotly Express.
460:Here is an example of enabling the x and y axis lines, and customizing their width and color, for a faceted histogram created with Plotly Express.
479:Here is an example of mirroring the x and y axis lines in a faceted histogram created using Plotly Express.
498:Here is an example of customizing the grid line width and color for a faceted scatter plot created with Plotly Express
539:Here's an example of manually specifying the x and y axis range for a faceted scatter plot created with Plotly Express.
698:Here is an example of reversing the direction of the y axes for a faceted scatter plot created using Plotly Express.
721:Here is an example of manually setting the reversed range of the y axes in a faceted scatter plot figure created using Plotly Express.
768:Here is an example of configuring a faceted scatter plot created using Plotly Express to always include zero for both the x and y axes.

./julia/legend.md
30:The fact that legend items are linked to traces means that when using [discrete color](/julia/discrete-color/), a figure must have one trace per color in order to get a meaningful legend. [Plotly Express has robust support for discrete color](/julia/discrete-color/) to make this easy.

plotlyjs.jl-docs on  issue-docs [$!?] via ஃ v1.6.1 via 🐍 v3.8.5 on ☁️  spencer.lyon@valorumdata.com 
❯ rg -S 'plotly.express' ./julia
./julia/text-and-annotations.md
44:import plotly.express as px

./julia/axes.md
177:Here is an example that configures the font family, size, and color for the axis titles in a figure created using Plotly Express.
204:Here is an example of turning on inside x-axis and y-axis ticks in a faceted figure created using Plotly Express. Note how the `yaxis_col` argument to `Layout` is used to only turn on the y-axis ticks for the left-most subplot.
225:Here is an example of updating the y-axes of a figure created using Plotly Express to display approximately 20 ticks.
253:Here is an example of updating the y axis of a figure created using Plotly Express to position the ticks at intervals of 0.5, starting at 0.25.
280:Here is an example of setting the exact location of ticks on the y axes of a figure created using Plotly Express.
307:Here is an example of enabling and styling the tick marks of a faceted figure created using Plotly Express. Note how the `col` argument to `yaxis attr` is used to only turn on and style the y-axis ticks for the left-most subplot.
332:Here is an example of disabling tick labels in all subplots for a faceted figure created using Plotly Express.
357:Here is an example of rotating the x-axis tick labels by 45 degrees, and customizing their font properties, in a faceted histogram figure created using Plotly Express.
460:Here is an example of enabling the x and y axis lines, and customizing their width and color, for a faceted histogram created with Plotly Express.
479:Here is an example of mirroring the x and y axis lines in a faceted histogram created using Plotly Express.
498:Here is an example of customizing the grid line width and color for a faceted scatter plot created with Plotly Express
539:Here's an example of manually specifying the x and y axis range for a faceted scatter plot created with Plotly Express.
698:Here is an example of reversing the direction of the y axes for a faceted scatter plot created using Plotly Express.
721:Here is an example of manually setting the reversed range of the y axes in a faceted scatter plot figure created using Plotly Express.
768:Here is an example of configuring a faceted scatter plot created using Plotly Express to always include zero for both the x and y axes.
806:Using `facet_col` from `plotly.express` let [zoom](https://help.plotly.com/zoom-pan-hover-controls/#step-3-zoom-in-and-zoom-out-autoscale-the-plot) and [pan](https://help.plotly.com/zoom-pan-hover-controls/#step-6-pan-along-axes) each facet to the same range implicitly. However, if the subplots are created with `make_subplots`, the axis needs to be updated with `matches` parameter to update all the subplots accordingly.

./julia/figure-labels.md
53:import plotly.express as px

./julia/legend.md
30:The fact that legend items are linked to traces means that when using [discrete color](/julia/discrete-color/), a figure must have one trace per color in order to get a meaningful legend. [Plotly Express has robust support for discrete color](/julia/discrete-color/) to make this easy.
65:import plotly.express as px
101:import plotly.express as px
169:import plotly.express as px
202:import plotly.express as px
255:import plotly.express as px
310:import plotly.express as px

plotlyjs.jl-docs on  issue-docs [$!?] via ஃ v1.6.1 via 🐍 v3.8.5 on ☁️  spencer.lyon@valorumdata.com 
❯ rg -S 'fig.show' ./julia
./julia/axes.md
218:fig.show()

./julia/text-and-annotations.md
57:fig.show()

./julia/legend.md
71:fig.show()
108:fig.show()
175:fig.show()
215:fig.show()
269:fig.show()
334:fig.show()
629:fig.show()

./julia/figure-labels.md
63:fig.show()
```

@tlyon3 please take a look and ping me for review once these issues are fixed.

**Comment by nicolaskruchten** [comment]
@sglyon in principle the CI job should fail if there is Python present in a `julia` block right?

**Comment by sglyon** [comment]
yes, I can't seem to see any CI status here. Any idea why we aren't getting reporting on the GH side?

**Comment by nicolaskruchten** [comment]
I'm still at a loss for why the CircleCI/Github integration isn't working. I filed a support ticket with Circle but no answer yet. Will keep at it.

**Comment by nicolaskruchten** [comment]
Try merging master into this branch, or rebasing? I think CircleCI is now hooked up, or at least it worked in my PR #6

**Pull request merged**